src/scala/org/pantsbuild/zinc/compiler/Main.scala

```diff
@@ -224,8 +303,10 @@ object Main {
 
       // post compile merge dir
       if (settings.postCompileMergeDir.isDefined) {
-        IO.copyDirectory(new File(settings.postCompileMergeDir.get.toURI),
-                         new File(settings.classesDirectory.toURI))
+        IO.copyDirectory(
```

Comment:
I'm surprised this succeeds unconditionally, and yet the function signature indicates to me that it does: https://www.scala-sbt.org/1.x/api/sbt/io/IO$.html#copyDirectory(source:java.io.File,target:java.io.File):Unit